lib/elixir_search_extractor_web/controllers/user_registration_controller.ex

```diff
@@ -0,0 +1,30 @@
+defmodule ElixirSearchExtractorWeb.UserRegistrationController do
+  use ElixirSearchExtractorWeb, :controller
+
+  alias ElixirSearchExtractor.Accounts
+  alias ElixirSearchExtractor.Accounts.User
+  alias ElixirSearchExtractorWeb.UserAuth
+
+  def new(conn, _params) do
+    changeset = Accounts.change_user_registration(%User{})
+    render(conn, "new.html", changeset: changeset)
+  end
+
+  def create(conn, %{"user" => user_params}) do
+    case Accounts.register_user(user_params) do
+      {:ok, user} ->
+        #        {:ok, _} =
+        #          Accounts.deliver_user_confirmation_instructions(
+        #            user,
+        #            &Routes.user_confirmation_url(conn, :confirm, &1)
+        #          )
```

Comment:
This is because we don't do the registration confirmation yet, right? 🤔

Reply:
@rosle you are right. It's coming in the next PR 😄